Library/Formula/libagar.rb

```diff
@@ -0,0 +1,37 @@
+class Libagar < Formula
+  desc "Cross-platform GUI toolkit"
+  homepage "http://libagar.org/"
+  url "http://stable.hypertriton.com/agar/agar-1.4.1.tar.gz"
+  sha256 "b0e62b754f134c3c0dd070a4fa62fa552654356eebab3d55e32d5d9b151a275e"
+  head "http://dev.csoft.net/agar/trunk", :using => :svn
+
+  depends_on "sdl"
+  depends_on "freetype"
+  depends_on "jpeg"
+  depends_on "libpng"
+
+  def install
+    ENV.deparallelize
```

Comment:
Can we report this upstream somewhere please and include the bug report above this line as a comment? Thanks!